public/index.html

```diff
@@ -54,6 +54,33 @@
         <path fill="#FFF" d="M18 2L2 2 2 18 18 18z" transform="translate(5 5)"/>
         <path d="M12 13v2H5v-2h7zm3-4v2H5V9h10zm0-4v2H5V5h10z" transform="translate(5 5)"/>
       </symbol>
+      <symbol id="icon-menu" viewBox="0 0 32 32">
+        <title>menu</title>
+        <path d="M4 24h24v-2.667H4V24zm0-6.667h24v-2.666H4v2.666zm0 0h24v-2.666H4v2.666zM4 8v2.667h24V8H4z"/>
+      </symbol>
+      <symbol id="icon-account-owner" viewBox="0 0 32 32">
+        <title>account-owner</title>
+        <path fill="#FFF" d="M16 2.667C8.64 2.667 2.667 8.64 2.667 16S8.64 29.333 16 29.333 29.333 23.36 29.333 16 23.36 2.667 16 2.667z"/>
+        <path fill="#FF8415" d="M16 25.6c-3.333 0-6.28-1.707-8-4.293.04-2.654 5.333-4.107 8-4.107 2.653 0 7.96 1.453 8 4.107-1.72 2.586-4.667 4.293-8 4.293zM16 6.667c2.213 0 4 1.786 4 4 0 2.213-1.787 4-4 4s-4-1.787-4-4c0-2.214 1.787-4 4-4z"/>
+      </symbol>
+      <symbol id="icon-close-menu" viewBox="0 0 32 32">
+        <title>close</title>
+        <path d="M8.458 25.428l16.97-16.97-1.886-1.886-16.97 16.97 1.886 1.886zM6.572 8.458l16.97 16.97 1.886-1.886-16.97-16.97-1.886 1.886z"/>
+      </symbol>
+      <symbol id="icon-help" viewBox="0 0 20 20">
+        <title>help</title>
+        <path d="M0 0L20 0 20 20 0 20z"/>
+        <path fill="#0592AF" d="M10 1.667C5.4 1.667 1.667 5.4 1.667 10S5.4 18.333 10 18.333 18.333 14.6 18.333 10 14.6 1.667 10 1.667zm.833 14.166H9.167v-1.666h1.666v1.666zm1.725-6.458l-.75.767c-.6.608-.975 1.108-.975 2.358H9.167v-.417c0-.916.375-1.75.975-2.358l1.033-1.05c.308-.3.492-.717.492-1.175 0-.917-.75-1.667-1.667-1.667s-1.667.75-1.667 1.667H6.667c0-1.842 1.491-3.333 3.333-3.333s3.333 1.491 3.333 3.333c0 .733-.3 1.4-.775 1.875z"/>
+      </symbol>
+      <symbol id="icon-download" viewBox="0 0 20 20">
+        <title>download</title>
+        <path d="M0 0L20 0 20 20 0 20z"/>
+        <path fill="#0592AF" d="M16.125 8.367C15.558 5.492 13.033 3.333 10 3.333 7.592 3.333 5.5 4.7 4.458 6.7 1.95 6.967 0 9.092 0 11.667c0 2.758 2.242 5 5 5h10.833c2.3 0 4.167-1.867 4.167-4.167 0-2.2-1.708-3.983-3.875-4.133zm-1.958 2.466L10 15l-4.167-4.167h2.5V7.5h3.334v3.333h2.5z"/>
+      </symbol>
+      <symbol id="icon-print" viewBox="0 0 20 20">
+        <path d="M0 0L20 0 20 20 0 20z"/>
```

Comment:
Same - you'll get a better visual result if you delete this line that adds a background square.